src/api/common/schemas/remote-options.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "remote-options",
+  "type": "object",
+  "properties": {
+    "trace": {"type": "boolean"},
+    "servers": {"type": "array", "items": {"type": "string"}}
```

Comment:
I think there is a JSON schema "format" option that you can use for URLs. Otherwise you can make a custom format.